lib/redis.js

```diff
@@ -21,15 +21,15 @@ module.exports = CoreObject.extend({
       }
 
       if (options.database) {
-        redisOptions.database = options.database;
+        redisOptions.db = options.database;
```

Comment:
I left the API the same here, but `then-redis` was pretty much the only library to call this parameter `database` instead of `db`.